theme/gatsby-browser.js

```diff
@@ -1,6 +1,9 @@
 import 'prismjs/themes/prism-solarizedlight.css'
 import 'prismjs/plugins/line-numbers/prism-line-numbers.css'
 
+import '@fontsource/crimson-text'; // Defaults to weight 400
+import '@fontsource/crimson-text/700.css'; // Weight 700 (Bold)
```

Comment:
These fonts must be what increased the document size, because I'm including them in my browser bundle. If they are too large, I may need to defer loading the fonts, because I don't want my browser bundle to 4x.

Reply:
I've pushed in a commit replacing these imports with **gatsby-plugin-google-fonts** to  leverage the built-in support for web fonts and serve them from a CDN like Google Fonts.

Reply:
Now it failed with a much smaller increase. I'm OK with this; I'll update the lighthouse config to accept up to 75k.

```
Expected <= 61440, but found 64816
```